.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+# Description
+Briefly explain what all changes you have made. Don't forget to mention any dependencies that are added in this change.
+
+Fixes # (issue number)
+
+## Type of change
+Please delete options that are not relevant.
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Checklist:
```

Comment:
A few more checklist points here - 
[ ] - The branch I'm pushing the code from is up-to-date with develop branch. 
[ ] - The title of this Pull Request is in accordance with this format - `Fix #IssueNumber: Small title summarising the change`. 
[ ] - The branch name follows the prescribed format of - `title-of-bug-fixed-or-feature-added`